apps/test/util/testUtils.js

```diff
@@ -219,3 +221,32 @@ exports.assertOwnProperty = function (obj, propertyName) {
 exports.debugMode = function () {
   return location.search.substring(1).split('&').indexOf('debug') !== -1;
 };
+
+/**
+ * jQuery.simulate was having issues in phantom, so I decided to roll my own
+ * drag simulation. May belong in a util file.
+ * @param {string} type
+ * @param {number} left Horizontal offset from top left of visualization to drop at
+ * @param {number} top Vertical offset from top left of visualization to drop at
+ */
+exports.dragToVisualization = function (type, left, top) {
+  // drag a new screen in
```

Comment:
Yep. Was originally just screen, but currently use this with at least SCREEN and BUTTON. Will update comment